sdk/maps/Azure.Maps.Geolocation/src/Models/IpAddressToLocationResult.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using Azure.Core;
+using Azure.Maps.Geolocation.Models;
+
+namespace Azure.Maps.Geolocation
+{
+    /// <summary> This object is returned from a successful call to IP Address to country/region API. </summary>
+    [CodeGenModel("IpAddressToLocationResult")]
+    public partial class IpAddressToLocationResult
+    {
+        /// <summary> Initializes a new instance of IpAddressToLocationResult. </summary>
+        internal IpAddressToLocationResult()
+        {
+        }
+
+        /// <summary> Initializes a new instance of IpAddressToLocationResult. </summary>
+        /// <param name="countryRegion"> The object containing the country/region information. </param>
+        /// <param name="ipAddress"> The IP Address of the request. </param>
+        internal IpAddressToLocationResult(CountryRegion countryRegion, string ipAddress)
+        {
+            CountryRegion = countryRegion;
+            IpAddress = ipAddress;
+            IsoCode = countryRegion.IsoCode;
+        }
+
+        /// <summary> The object containing the country/region information. </summary>
+        [CodeGenMember("CountryRegion")]
+        internal CountryRegion CountryRegion { get; }
+        /// <summary> The IP Address&apos;s 2-character code [(ISO 3166-1)](https://www.iso.org/iso-3166-country-codes.html) of the country or region. Please note, IP address in ranges reserved for special purpose will return Null for country/region. </summary>
```

Comment:
Usually we use `<see href="<url>">xxx</see>` to replace `[xxx](<url>)` in C# docstring